### PR TITLE
[GraphQL] Add documentation about serialization groups

### PR DESCRIPTION
**Pull request by alanpoulain**
Because it can be really useful!

### CODE REVIEW

File: core/graphql.md [hunk 1]
```diff
@@ -174,3 +174,62 @@ class Book
     // ...
 }
 ```
+
+## Serialization Groups
+
+You may need to prevent clients using your GraphQL endpoint to access some attributes of your resources.
```
Comment (soyuka) [inline core/graphql.md]: Not sure to understand this sentence. What about:

You may want to restrict some resource's attributes to your clients with GraphQL.

File: core/graphql.md [hunk 2]
```diff
@@ -174,3 +174,62 @@ class Book
...
+
+Like described in the [serialization process](serialization.md) documentation, you can use serialization groups to expose only the attributes you want in queries or in mutations.
```
Comment (soyuka) [inline core/graphql.md]: Following the previous sentence:

As described in the serialization process documentation, ...

File: core/graphql.md [hunk 3]
```diff
@@ -174,3 +174,62 @@ class Book
...
+
+If the (de)normalization context between GraphQL and REST is different, use the `graphql` key to change it.
+
+A query is only using a normalization context whereas a mutation is using a denormalization context for the input and a normalization context for its output.
```
Comment (soyuka) [inline core/graphql.md]: Note that:

- A **query** is using the normalization context
- A **mutation** is using the denormalization context for it's input and the normalization context for it's output

File: core/graphql.md (hunk 2 above)
Comment (alanpoulain) [reply core/graphql.md]: What do you mean by: "Following the previous sentence:"?

Comment (soyuka) [reply core/graphql.md]: You may need ... 

As described in the ....

**Comment by Simperfit** [comment]
Thanks Alan!

**Pull request merged**